ChangeLog

```diff
@@ -36,6 +36,11 @@ Changes
    * Clarify ECDSA documentation and improve the sample code to avoid
      misunderstandings and potentially dangerous use of the API. Pointed out
      by Jean-Philippe Aumasson.
+   * Add support for alternative implementation for ECDH, controlled by new
```

Comment:
I am sorry, I've missed this at the first pass: could we put this in the "Features" section instead of among the "Changes"?